src/azure-cli/azure/cli/command_modules/eventgrid/_params.py

```diff
@@ -198,10 +204,13 @@ def load_arguments(self, _):    # pylint: disable=too-many-statements
         c.argument('setup_uri', help="URI of the partner website that can be used by Azure customers to setup Event Grid integration on an event source.")
         c.argument('partner_registration_id', help="The fully qualified ARM Id of the partner registration that should be associated with this partner namespace. This takes the following format: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventGrid/partnerRegistrations/{partnerRegistrationName}.")
         c.argument('partner_topic_source', arg_type=partner_topic_source_type)
-        c.argument('desination_topic_name', help="Name of the partner topic associated with the event channel.")
+        c.argument('destination_topic_name', help="Name of the partner topic associated with the event channel.")
```

Comment:
With this argument dest renamed, we should rename where it is used too. See Line 414 and Line 424
https://github.com/Azure/azure-cli/blob/983cb41d4cfce51fd07e7e0244a6383af35dd265/src/azure-cli/azure/cli/command_modules/eventgrid/custom.py#L406-L424

Reply:
done locally. thanks!